big_tests/tests/graphql_vcard_SUITE.erl

```diff
@@ -141,6 +156,32 @@ user_get_others_vcard_no_user(Config, Alice) ->
     Result = user_get_vcard(Alice, <<"AAAAA">>, Config),
     ?assertEqual(<<"User does not exist">>, get_err_msg(Result)).
 
+%% Domain admin test cases
+
+domain_admin_set_vcard_no_user(Config) ->
+    Vcard = complete_vcard_input(),
+    get_unauthorized(set_vcard(Vcard, <<"AAAAA">>, Config)).
+
+domain_admin_get_vcard_no_user(Config) ->
+    get_unauthorized(get_vcard(<<"AAAAA">>, Config)).
+
+domain_admin_get_vcard_no_permission(Config) ->
+    escalus:fresh_story_with_config(Config, [{alice_bis, 1}],
+                                    fun domain_admin_get_vcard_no_permission/2).
+
+domain_admin_get_vcard_no_permission(Config, AliceBis) ->
+    Result = get_vcard(user_to_bin(AliceBis), Config),
```

Comment:
This is what I meant in the comments above - test a user at the secondary domain.